examples/nlp/language_modeling/conf/megatron_t5_prompt_learning.yaml

```diff
@@ -12,6 +12,7 @@ trainer:
   max_steps: -1
   log_every_n_steps: 10
   val_check_interval: 1.0
+  accumulate_grad_batches: 1
```

Comment:
I am comparing my branch (https://github.com/NVIDIA/NeMo/pull/4920) to yours. From my research, this config should be removed. Could you please confirm?

Reply:
It can be removed. It's not hurting anything having it there though.

Reply:
ok. i will keep what `main` did then.